network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube_regrets_page.html

```diff
@@ -87,7 +87,3 @@
 {% block footer_block %}
 {% include "partials/footer.html" %}
 {% endblock %}
-
-{% block extra_scripts %}
-  <script src="{% static "_js/youtube-regrets.compiled.js" %}"></script>
```

Comment:
This was a bad move. This script needs to wait for various functions and calls in `main.js` to finish first.

Reply:
Hm, does it? I thought it was only dependent on the DOM being done, in which case we're only missing the `defer` attribute.